port_ocean/tests/core/handlers/entity_processor/test_jq_entity_processor.py

```diff
@@ -0,0 +1,228 @@
+from typing import Any
+from unittest.mock import AsyncMock, Mock
+import pytest
+
+from port_ocean.context.ocean import PortOceanContext
+from port_ocean.core.handlers.entity_processor.jq_entity_processor import (
+    JQEntityProcessor,
+)
+from port_ocean.core.ocean_types import CalculationResult
+from port_ocean.exceptions.core import EntityProcessorException
+
+
+@pytest.mark.asyncio
+class TestJQEntityProcessor:
+
+    @pytest.fixture
```

Comment:
Oh nice, closure automagically since it's in the test class?
If so, good practice that's worth sharing.